packages/components-organizations-registrar/src/pages/organizations/components/OrganizationCreateTitle.jsx

```diff
@@ -0,0 +1,20 @@
+import { Typography } from '@mui/material';
+import { useConfig } from '@/utils/ConfigContext';
+
+export const OrganizationCreateTitle = () => {
```

Comment:
im ok witha  component for this, but im struggling why this is defined in a separate file when it could be in the one and only file it is used. The same applies to OrganizationSubmitButton.

Reply:
@sloops77 I was trying to make the file a little bit more readable by moving components out of the main flow file. useConfig is needed only for titles.